aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -103,11 +119,15 @@ class QConvUnpackWeightsInt8 final : public c10::OperatorKernel {
   }
 };
 
-static auto registry = c10::RegisterOperators().op(
-    "quantized::conv_unpack(Tensor packed_weights)"
-    " -> (Tensor unpacked_weights, Tensor? B_origin)",
-    c10::RegisterOperators::options().kernel<QConvUnpackWeightsInt8>(
-        TensorTypeId::CPUTensorId));
+static auto registry =
+    c10::RegisterOperators()
+        .op("quantized::conv_unpack(Tensor packed_weights)"
```

Comment:
Same comment on naming inconsistency as above

Reply:
Same as the comments above, because of backward compatibility tests, we cannot change the existing op names and args

Reply:
What I’m saying above is we probably *should* break backwards compatibility, otherwise we’re in a state that doesn’t make sense. We can defer this to another PR though for now